### PR TITLE
Make kraken/sunspider-conpare-results work.

### PR DESCRIPTION
**Pull request by lazyparser**
Fixed the 'die at Line 110' error.

### CODE REVIEW

File: benchmarks/kraken/sunspider-compare-results [hunk 1]
```diff
@@ -55,7 +55,7 @@ GetOptions('shell=s' => \$jsShellPath,
 $suite = "ubench" if ($ubench);
 $suite = "v8-v4" if ($v8suite);
 $suite = "parse-only" if ($parseOnly);
-$suite = "sunspider-0.9.1" if (!$suite);
+$suite = "kraken-1.1" if (!$suite);
```
Comment (h4writer) [inline benchmarks/kraken/sunspider-compare-results]: at line 41 there is:
--suite           Select a specific benchmark suite. The default is sunspider-0.9.1

Can you also update that?

**Comment by h4writer** [comment]
Can you also do this for the "misc" benchmark dir?

**Comment by lazyparser** [comment]
> at line 41 there is:
> --suite Select a specific benchmark suite. The default is sunspider-0.9.1
> Can you also update that?
Fixed.

> Can you also do this for the "misc" benchmark dir?
Sure. I'd submit another PR.

**Comment by lazyparser** [comment]
> Can you also do this for the "misc" benchmark dir?

see https://github.com/h4writer/arewefastyet/pull/90

**Comment by h4writer** [comment]
Thanks!

**Pull request merged**